docs/guide/controllers.md

```diff
@@ -399,15 +407,15 @@ In addition to creating a class for a `QueryType`, you must also register the Qu
 #### By convention
 
 Any class named `<Bundle>\QueryType\*QueryType` that implements the QueryType interface will be registered as a custom QueryType.
-Example: `AppBundle\QueryType\LatestContentQueryType`.
+Example: `AcmeExample\QueryType\LatestContentQueryType`.
```

Comment:
If you are still referring to a bundle you should use `*Bundle` in the namespace to avoid confusion :) So `AcmeExampleBundle` is a better name. With our default setup you can create bundle folder under `src/` directory:
```
- src/
    - AcmeExampleBundle/
        - Resources/
            - (...)
        - AcmeExampleBundle.php
```

And it will be available under `App\AcmeExampleBundle\AcmeExampleBundle` classname.